src/payment/data/reducers.js

```diff
@@ -57,8 +60,34 @@ const basket = (state = basketInitialState, action = null) => {
   return state;
 };
 
+const captureContextInitialState = {
+  isCaptureKeyProcessing: false,
+  flexMicroformEnabled: false,
```

Comment:
I'm thinking we don't actually need to track this in the state since it's always overridden by the query string parameter in the selector